remotespark/livyclientlib/exceptions.py

```diff
@@ -0,0 +1,34 @@
+class LivyClientLibException(Exception):
```

Comment:
What's the model for Exceptions for this library after this change? Should a developer never raise an exception purposefully that is not a `LivyClientLibException`?

Can we document it in this class?